.github/CONTRIBUTING.md

```diff
@@ -40,7 +40,32 @@ for more information about PR evaluations.
 
 ## Adding data
 
-You can create a sub-dataset in the CONP repository as follows:
+You can create a sub-dataset in the CONP repository in two ways:
+
+1. Using Zenodo
+    1. Upload your dataset to https://zenodo.org with the specific keyword `canadian-open-neuroscience-platform` if the dataset is less than 50GB
```

Comment:
Why less than 50GB?  (I know that Zenodo has a dataset file limit but do others?)

Reply:
![image](https://user-images.githubusercontent.com/35869923/70544400-0427ed00-1b3a-11ea-9987-62cc723486ea.png)
It seems like we can get more than 50GB but we would need to contact them

Reply:
I updated the documentation to reflect that option